app/repository/pg/user.go

```diff
@@ -29,8 +30,7 @@ func NewRepo(db *sql.DB) *Repo {
 func (r *Repo) Add(user entities.User) error {
 	const sqlStatement = `INSERT INTO "user" (first_name, last_name, email, password)
 						  VALUES ($1, $2, $3, $4)`
-
-	if _, err := r.DB.Exec(sqlStatement, user.FirstName, user.LastName, user.Email, user.Password); err != nil {
+	if _, err := r.DB.Exec(sqlStatement, user.FirstName, user.LastName, user.Email, hash.Hash(user.Password)); err != nil {
```

Comment:
I don't really like this hash.Hash() naming...
What about hash.FromString() or hash.SHA256()... maybe we will be lucky to find a good name 😄

Reply:
Make sense. Changed the name of the function to SHA256